### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by Amazinite**
Taking a crack at updating the changelog, as [it's been a while.](https://github.com/endless-sky/endless-sky/commit/bff93bda46a93d802aac4b484a97a8759c736903) 

Updates the changelog up to: b73d640aa3285b794c179086f7cff454234b14ad

<details><summary>Commits that I didn't put on the changelog, mainly because I didn't know what to put down or where to put it. Mostly stuff from @tehhowch, so I suggest he update the changelog with this stuff.</summary>
	5b4cca360ee3581ba660b2edbc9fc6ed0e4f5c2e
	6742b03db4d274e91fdbbd36d5555da03d7ca099
	726cd23dc638b888ec25acded1288cb2e45eb09d
	4a95c12215b70115f27d92d419380f7a117d9e9d
	9c900de7e053ab34f1e72d2480ede8f46c024142
	b2d917ed82ed821ad77058d39243813b35849b78
	e6cc89a233a2fc200ae971b9ffade21c232a0c6b
	91c45e7207e836be8ec73b5306135c243423b204
	330225ec62b8c1d98edc34e6375cc138076daba2
	d3d86201bcc464799a6c6a31bc95b7dc42297f0b
	1730dc4bf1017e4e254af2600bfdc08f9f7f6e67
	cf2b2924fce3bc5882ca90fbb4121ec51f7278a7
	377b744f461150404abc2ffb13460c9fcaef8144
	edecc3063c24341f823a59ed5cabe04c4a31c7d5
	79af6a66568b9aca63a09ee6c1492dc19fcc726f
	7646e3095a3aa5db8cf5e4749c7b967c61a42bb4
	d38ff922e2446778b9d0b923dbaf7f82d2e04a1d
	5f538c1ad06876a3e89cb49302fb86f6b90b4cf0
	411c1d3557995b69d247b25808730dd53039bd1d
	c5fcb7186fe5e5e0edd5018aa08f7efea7b13e6a
	ef6cac55dbbccfe4a8493a1a89030d9d53a6e652
	4aa9f4cece9b22d554f04962c94d6fa1d09497d7
	0eefe05967ca4a02974da50146cd174435e1c896
	c50bb8bd51c07ebbd158e0ac23377b681039566e
	5ef61c5f463d2c793ec15f79cc38318868c6fb38
	f142325857e2e110d5b30cbe9e30e538399c45a9
	346faa4d250e6d250d243817710639386019655c
	dccf48940caee0beb533ae8097b2deb49f535c2f
	b8089bfa88d8c04a6cf2b3bbf3bbb34b9546bdbb
	53fe39a7da07ae444a3e20352272657a3d65ede1
	abc0efde71d5c193ad3ae06c207a1fd84e54286f
	d717801604fe59f48313befa31f81d94233e22d8
	2c251285488ff2689645eb6f6bc468a01c6dc28e
	dca8ecb2c179194393bf27ee2de6dc90b5085f66
	7d1301faedcb959fb5e14fadb53605357bfb9a36
	a769c110cdbf9a9568a692b1c3f4e44212c835a3
	bb5ec746f3a7e8583a07e04982a2c1aa32f3f4cc
	6e7043e8363c468feea1990e7e5ede0b337c4a3e
	54e27db9da1391d407b02b2baee0f38635e5c470
	8e204e9ba185a2f0594006ba8143e8621714e635
	6dc743e3217756e2d031a231af05dc8a8e06076f
	1df09fcf23afbe81d2729e17be163d9d4c2c83f8
	edb535f3c8e71b1ab7633782a9f213dc9595fb2f
	97b4c4a2e8ff56919bb6e64955aea45449d8212c
	5732b51d4232285c9be27fec6668511f699dcb77
	1fb48ecd1cf496885e7fd9e30b95e14ee0e328d9
	bd071c3aae2359df028adfc5202a11bc6af2b977
	7882e00cd38204baab2201e646cecb95885e4c70
</details>

### CODE REVIEW

**Comment by Amazinite** [comment]
Should the Bactrian and Remnant stuff go under big changes?

**Comment by MCOfficer** [comment]
they are the biggest story changes in this log, so i'd say yes.

File: changelog [hunk 1]
```diff
@@ -106,6 +145,12 @@ Version 0.9.9:
       * Added more fancy passenger ships in the Paradise systems. (@Pointedstick)
       * The cloaking device is now unplunderable. (@Bladewood)
       * Switched some landscape images so that fewer of them are used for multiple planets. (@Amazinite)
+      * All red wormholes now require the quantum keystone attribute to be used. (@Nescio0)
+      * Wasps will now spawn in small Core pirate fleets. (@Nescio0)
+      * Test dummy ships now only aim to disable instead of destroying. (@tehhowch, @Amazinite)
+      * Command centers are now unplunderable. (@AlexBassett)
+      * Navy ships, human medium warships, and human heavy warships now come equipped with hand to hand weapons. (@Fzzr)
```
Comment (Fzzr) [inline changelog]: Update after #4216: 
`* Navy Ships, human medium warships, human heavy warships, human heavy freighters, and certain human variants now come equipped with hand to hand weapons. (@Fzzr)`

**Comment by Pointedstick** [comment]
I wonder if we should set a new policy that every PR must also update the changelog at the same time. That might make this much easier over time.

**Comment by Amazinite** [comment]
Yeah, I was thinking about that.

**Comment by jafdy** [comment]
I was thinking about it too, I figured that adding the requirement of a sensible change log entry to a PR would lead to more steps to merge and create an inconsistent style, I think this way is better.

**Comment by tehhowch** [comment]
Another issue with requiring PRs to also update the changelog is that not everything *should* go into the changelog--we already have the commit history for that.

**Pull request merged**